README.md

```diff
@@ -516,6 +524,36 @@ Leave a blank line between a block and the next statement.
   return obj;
 ```
 
+Each line of code should be no more than 100 characters. This helps with readability in an IDE as well as in pull requests. This includes all lines including strings, template literals, and comments.
+
+```javascript
+  // bad
+  const amazingPeople = { mathematicians = ['Katherine Johnson', 'Dorothy Vaughan'], scientists: ['Alice Ball', 'Mary Elliott Hill'], astronauts: ['Mae C. Jemison', 'Stephanie Wilson']};
+
+  // good
+  const amazingPeople = {
+    mathematicians = ['Katherine Johnson', 'Dorothy Vaughan'],
+    scientists: ['Alice Ball', 'Mary Elliott Hill'],
+    astronauts: ['Mae C. Jemison', 'Stephanie Wilson']
+  };
+
+  // bad
+  // This is a really long comment that should really be written in multiple lines but instead has been written in a single line.
+
+  // good
+  // This is a really long comment that has been written on multiple lines instad of on a single
+  // line so that it is easier to read.
+
+  // bad
+  const str = `this is a really long string that should really written over multiple lines but instead has been written in a single line`;
+
+  // good
+  const str = `this is a really long string that has been written over multiple lines instead
+  of being written in a single line`;
+```
+
+Each function should have no more than 200 lines to reduce complexity and increase testability. This is especially important in React components where the render functions can quickly get out of hand trying to do too much.  This includes all comments and whitespace but ignores IIFEs (Immediately Invoked Function Expressions) to match SonarQube functionality.
```

Comment:
Maybe I'm missing something, but line 555 is referring to lines of code, but the above examples are referring to length of lines.

Reply:
Yes, this is a new section not related to the example above.  I thought adding an example for it would be overkill as it would need to be 200 lines long :)